src/localization/en.json

```
@@ -421,6 +421,9 @@
         "average_playtime": "Average playtime",
         "average_playtime_2weeks": "Average playtime in the last two weeks",
         "formatted_time": "__hours__ hours, __minutes__ minutes"
+        "playtime_unit_day": "__days__d"
+        "playtime_unit_hour": "__hours__h"
```

Need trailing commas here apart from the last item otherwise it's invalid JSON
```suggestion
        "formatted_time": "__hours__ hours, __minutes__ minutes",
        "playtime_unit_day": "__days__d",
        "playtime_unit_hour": "__hours__h",
```